Dockerfile

```diff
@@ -17,22 +17,27 @@ RUN apk --no-cache add \
     && adduser -u 1000 -D -h $PROJECT_ROOT sw6 sw6 \
     && rm /etc/nginx/conf.d/default.conf
 
+# Install compose
+RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/bin --filename=composer
```

Comment:
```suggestion
COPY --from=composer /usr/bin/composer /usr/bin/composer
```

Reply:
Maybe we could use composer:2. The install step would be much faster

Reply:
@shyim changed as proposed, tested and changed to composer v2